roles/pulp-api/README.md

```diff
@@ -1,12 +1,17 @@
 Pulp API
 ========
 
-A role to setup Pulp 3's API service.
+A role to setup Pulp 3's API:
```

Comment:
A role to setup Pulp 3 API

Reply:
IMO, this variation would need an article "A role to setup the Pulp 3 API". 
Should be set up also. I should have caught that earlier. 
The possessive isn't wrong either. I read a lot about this in my previous job. It's acceptable now in technical documentation.

Reply:
articles are one of my weakest spots so I'm tottally trusting you :D

Reply:
articles are one of my weakest spots so I'm totally trusting you :D

Reply:
re: possesive, I know this is not wrong, It is just improves the readability because there is no `3's`